web/style.css

```diff
@@ -23,6 +28,16 @@
 	--palette-7: #e796ea; /* purple */
 }
 
+@font-face {
+font-family: "Graphik";
+src: url(assets/fonts/GraphikRegular.otf);
+}
```

Comment:
Please, remove these lines. See also https://github.com/SpiderOak/webwormhole/pull/7#issuecomment-975576561

Reply:
The font has been removed.